ValkeyJSON.md

```diff
@@ -75,9 +75,15 @@ called KeyTable, which is a sharded hash table storing key tokens and reference
 
 ### Document Size Limit
 
-Having a limit on json key size is generally considered as a good design. Without the limit, a malicious program can
-repeatedly call "json.arrinsert" or similar commands to make a json key grow forever, leading to OOM. ValkeyJSON will 
-provide a module config - json.max-document-size - to set the size limit.
+We added a limit on JSON key size as a good design practice. Without the limit, a malicious program could repeatedly call 
+"json.arrinsert" or similar commands to make a JSON key grow indefinitely, potentially leading to an out-of-memory (OOM) condition.
+
+The ValkeyJSON module will provide a configuration option, json.max-document-size, that allows users to set a size limit for
+JSON key size.
+
+However, the default value of json.max-document-size will be set to -1, meaning the size of JSON key will be unlimited. 
```

Comment:
```suggestion
However, the default value of json.max-document-size will be set to 0, meaning the size of JSON key will be unlimited. 
```

Reply:
I think 0 is zero/disabled, `-1` is more logical as to indexes as `-1` also refers to last and here its infinite.